source/tutorial/rotate-log-files.txt

```diff
@@ -7,111 +7,76 @@ Rotate Log Files
 Overview
 --------
 
-Log rotation using MongoDB's standard approach archives the current
+When used with the :option:`--logpath` option or :setting:`systemLog.logPath` setting,
+:program:`mongod` and :program:`mongos` instances report
+a live account of all activity and operations to a log file. 
+When reporting activity data to a log file, by default, MongoDB only rotates logs
+in response to the :dbcommand:`logRotate` command, or when the
+:program:`mongod` or :program:`mongos` process receives a ``SIGUSR1``
+signal from the operating system.
+
+MongoDB's standard log rotation approach archives the current
 log file and starts a new one. To do this, the :program:`mongod` or
 :program:`mongos` instance renames the current log file by appending a
-UTC (GMT) timestamp to the filename, in :term:`ISODate` format. It then
+UTC timestamp to the filename, in :term:`ISODate` format. It then
 opens a new log file, closes the old log file, and sends all new log
 entries to the new log file.
 
-MongoDB's standard approach to log rotation only rotates logs
-in response to the :dbcommand:`logRotate` command, or when the
-:program:`mongod` or :program:`mongos` process receives a ``SIGUSR1``
-signal from the operating system.
+With the :option:`--syslog` option, :program:`mongod` and :program:`mongos`
+report all activity to the ``syslog`` rather than a log file.
```

Comment:
I think I actually meant to get rid of the Finally… paragraph, but I think I'll go with your suggestion. Jeez.